zebra/zebra_ns.h

```diff
@@ -34,6 +34,18 @@ struct nlsock {
 };
 #endif
 
+struct zebra_ns_tables {
```

Comment:
why the plural form? I think `zebra_ns_table` would make more sense since this structure represents a single table.

Reply:
fixed